Automattic/wp-calypso
# Goodbye, IE 11

### Changes proposed in this Pull Request

Enable redirect for fallback browsers in all environments. Follow-up from #53221, which implemented the redirect for development and some testing environments. I also took the opportunity to remove documentation referencing IE11 and the fallback build.

See #53222 for follow-up tasks.

### Testing instructions

Use calypso.live for the following:

- Verify that IE 11 is redirected to `/browsehappy`. Browserstack works well for a quick test. (Just load the calypso.live link in IE 11. You can also test the "continue anyways" path and ensure it does not redirect after that.
- Verify that supported browsers are not redirected. (Load calypso.live in something like chrome or firefox and ensure that you do not get redirected.

We will have to double check this behavior after shipping once the deployment makes it to staging, and then again after it gets to prod, as this code change modifies environment configuration.
